inlong-manager/manager-plugins/pom.xml

```diff
@@ -56,6 +53,9 @@
         </profile>
         <profile>
             <id>v1.15</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
```

Comment:
building plugin for all Flink versions is fine here, so, there is no need to make the 1.15 profile active by default

Reply:
There is a problem here. If I activate all versions, it will depend on the `sort-flink-dependencies-v1.13` module, which I have excluded because it is in the `sort-flink-v1.13` module. It also depends on `sort-format-json-v1.13`, but this module is excluded because the default activated profile has changed to v1.15.

Reply:
This modification breaks the management of the InLong manager.
For now, users only need to change the `flink.version` in `flink-sort-plugin.propertie` if want to change the version of Flink.
But with this modification, users need to rebuild the plugin modules and replace the `manager-plugins-flink-xx.jar`

Reply:
Yes, this modification does have an impact on management of the InLong manager.
I will review it again. But from the current situation, many modules in `sort-flink-v1.13` depend on `sort-format-json-v1.13` module which will be excluded by v1.15 profile activation.